Sources/Analytics/Errors/ErrorAnalytics.swift

```diff
@@ -3,4 +3,5 @@ import Logging
 
 enum ErrorAnalyticsScreen: String, LoggableScreen, NamedScreen {
     case generic = "genericErrorScreen"
+    case unableToLogin = "unableToLoginErrorScreen"
```

Comment:
Is there a reason you need to assign string values to these? Wouldn't `generic` and `unableToLogin` suffice and let the `String` rawValue do the work for you?

Reply:
Conceivably yes 🙂 we're just putting dummy values in for now as analytics work won't be for some time yet but yeah I get your drift